### PR TITLE
feat(components): add table component

### PR DESCRIPTION
**Pull request by helen-v**
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/table/Table.ts [hunk 1]
```diff
@@ -0,0 +1,128 @@
+import reset from "@/wc_scss/reset.scss";
```
Comment (yananym) [inline web-components/src/components/table/Table.ts:1]: please add the same disclaimer we have in other files

File: web-components/src/components/table/scss/module.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+@import "@/wc_scss/tools/functions/core";
```
Comment (yananym) [inline web-components/src/components/table/scss/module.scss:1]: add disclaimer here to. really, it should be in every file. for the purposes of this pr - add the disclaimer in every new file you are creating. both in component and sandbox

File: web-components/src/components/table/scss/table.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    }
+    th {
+      text-align: left;
+      font-size: 12px;
+      font-weight: bold;
+      color: #545454;
+      padding-left: 12px;
+
+      a {
+        color: #545454;
```
Comment (yananym) [inline web-components/src/components/table/scss/table.scss:18]: should not be hardcoded

File: web-components/src/components/table/Table.ts (hunk 1 above)
Comment (helen-v) [reply web-components/src/components/table/Table.ts:1]: Yes, it's just draft PR

File: web-components/src/components/table/scss/module.scss (hunk 2 above)
Comment (helen-v) [reply web-components/src/components/table/scss/module.scss:1]: Added

File: web-components/src/components/table/scss/table.scss (hunk 3 above)
Comment (helen-v) [reply web-components/src/components/table/scss/table.scss:18]: Of course, added tokens.

File: web-components/src/components/table/Table.ts [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    this.headerRow = this.results.data[0];
+    this.csvData = this.results.data.slice(1, this.results.data.length);
+  }
+
+  sortTab(ev: Event, key: any) {
+    const elCell = ev.target as HTMLTableElement;
+    const sortArr = Array.from(this.csvData); 
+    const index = this.headerRow.indexOf(key);
+
+    console.log(sortArr);
```
Comment (futurethang) [inline web-components/src/components/table/Table.ts:55]: nit:  cleanup console logs

Comment (helen-v) [reply web-components/src/components/table/Table.ts:55]: Thanks! Forgot it. Removed!

**Pull request merged**